cloud/aws/sqs/variables.tf

```diff
@@ -0,0 +1,193 @@
+# Global
+
+variable "environment" {
+  description = "Infrastructure environment"
+  type        = string
+}
+
+# SignalFx module specific
+
+variable "notifications" {
+  description = "Notification recipients list for every detectors"
+  type        = list
+}
+
+variable "prefixes" {
+  description = "Prefixes list to prepend between brackets on every monitors names before environment"
+  type        = list
+  default     = []
+}
+
+variable "filter_custom_includes" {
+  description = "List of tags to include when custom filtering is used"
+  type        = list
+  default     = []
+}
+
+variable "filter_custom_excludes" {
+  description = "List of tags to exclude when custom filtering is used"
+  type        = list
+  default     = []
+}
+
+variable "detectors_disabled" {
+  description = "Disable all detectors in this module"
+  type        = bool
+  default     = false
+}
+
+# AWS SQS detectors specific
+
+variable "heartbeat_disabled" {
+  description = "Disable all alerting rules for heartbeat detector"
+  type        = bool
+  default     = null
+}
+
+variable "heartbeat_notifications" {
+  description = "Notification recipients list for every alerting rules of heartbeat detector"
+  type        = list
+  default     = []
+}
+
+variable "heartbeat_timeframe" {
+  description = "Timeframe for system not reporting detector (i.e. \"10m\")"
+  type        = string
+  default     = "20m"
+}
+
+# Visible_messages detectors
+
+variable "visible_messages_disabled" {
+  description = "Disable all alerting rules for visible_messages detector"
+  type        = bool
+  default     = null
```

Comment:
should be true

Reply:
we want the detectors disabled by default?

Reply:
I thought we were setting "detectors_disabled" to false for all modules then the detector specific ones null by default

Reply:
monitors disabled by default from datadog side should remain disabled by default for signalfx too but yes by default we do like you said

Reply:
https://github.com/claranet/terraform-datadog-monitors/tree/master/cloud/aws/sqs#purpose

Reply:
done